.github/workflows/release-new-charts.yaml

```diff
@@ -8,16 +8,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: azure/setup-helm@v1
-        with:
-          version: 'v3.1.2'
-      - uses: broadinstitute/setup-chart-releaser@v1
-        with:
-          version: '1.0.0-beta.1'
       # Needed to actually find the last released tag.
       # https://github.com/helm/chart-releaser-action/issues/13
       - name: Fetch full git history
         run: git fetch --prune --unshallow
-      - run: ./hack/upload-changed-charts
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
```

Comment:
To check: where does GITHUB_ACTOR come from? It was shown verbatim in the action docs.

Reply:
Answer: https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables